google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1209,7 +1211,17 @@ public Builder setDefaultAcl(Iterable<Acl> acl) {
 
     @Override
     public Builder setLabels(Map<String, String> labels) {
-      this.labels = labels != null ? ImmutableMap.copyOf(labels) : null;
+      if (labels != null) {
+        this.labels =
+            Maps.transformValues(
+                labels,
+                new Function<String, String>() {
+                  @Override
+                  public String apply(String input) {
+                    return input == null ? "" : input;
```

Comment:
For consistency could you use `Data.<String>nullOf(String.class)`.

Reply:
Seems obtuse to me, but I'll do it for consistency. Is there a reason it wouldn't be better/more readable to do it the other way around and replace other instances of  Data.<String>nullOf(String.class) with "" for readability?

Reply:
Could you file an issue to better address this in the library for readability? So it's done all at once instead of a one-off and then we can maintain consistency for now.